tests/jax/test_jax_3d_eldry.py

```diff
@@ -0,0 +1,1005 @@
+"""
+
+Script for testing the accuracy of the elastic dry friction implementation 
+that uses JAX - Compares AFT results to those from vector Jenkins
+
+Tests include:
+    1. Low and high normal displacement checks against Jenkins with different
+    values of Fs
+    2. Verify second harmonic of normal induces second harmonic of
+    tangential force when slipping is present (tests 3 and 5)
+    3. Verify if separated that all is zero.
+
+"""
+
+# Standard imports
+import numpy as np
+import sys
+import unittest
+
+# Python Utilities
+sys.path.append('../..')
+
+# # vectorized (non JAX version)
+# from tmdsimpy.jax.nlforces.jenkins_element import JenkinsForce 
+
+# JAX version for elastic dry friction
+from tmdsimpy.jax.nlforces.elastic_dry_fric_3d import ElasticDryFriction3D 
+
+# JAX version for elastic dry friction
+from tmdsimpy.jax.nlforces.elastic_dry_fric_2d import ElasticDryFriction2D 
+
+import tmdsimpy.harmonic_utils as hutils
+from tmdsimpy.vibration_system import VibrationSystem
+
+
+sys.path.append('..')
+import verification_utils as vutils
+
+
+###############################################################################
+###     Testing Class                                                       ###
+###############################################################################
+
+def run_comparison(obj, Unl, w, h, Nt, force_tol, df_tol, eldry2d, 
```

Comment:
This function definition is confusing. Can you explicitly pass in a `eldry_2d` and `eldry_3d` objects and then compare them rather than using `obj.eldry_force`, which is not immediately clear what it is or the fact that it is getting passed in here from the function signature.